playbooks/openshift-master/private/config.yml

```diff
@@ -176,6 +176,18 @@
     openshift_no_proxy_etcd_host_ips: "{{ hostvars | lib_utils_oo_select_keys(groups['oo_etcd_to_config'] | default([]))
                                                 | lib_utils_oo_collect('openshift.common.ip') | default([]) | join(',')
                                                 }}"
+  pre_tasks:
+  # This will be moved into the control plane role once openshift_master is removed
```

Comment:
It would be good to have an external doc that tracks the when and where code should be moved/refactored based upon milestones.

Reply:
Will do

Reply:
https://docs.google.com/document/d/1uVRH_bd2K53aYbdqrFtwl3rF6uyUVJ-McFnGYvob4VM/edit